src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-form/rgw-user-form.component.html

```diff
@@ -655,10 +658,19 @@
                     i18n>This field is required.</span>
               <span class="invalid-feedback"
                     *ngIf="userForm.showError('bucket_quota_max_objects', frm, 'min')"
-                    i18n>The entered value must be >= 0.</span>
+                    i18n>Enter a positive number.</span>
             </div>
           </div>
         </fieldset>
+
+      <!-- Advanced Section -->
+      <cd-form-advanced-fieldset>
+        <!-- User Rate Limit -->
+        <rgw-rate-limit [type]="'user'"
+                        [isEditing]="this.editing"
+                        [id]="this.getUID()">
```

Comment:
function call in template is something [we should avoid if we can.](https://medium.com/showpad-engineering/why-you-should-never-use-function-calls-in-angular-template-expressions-e1a50f9c0496)

Is there a way this UID can be stored into a constant and we just have to pass that constant here? 
```suggestion
                        [id]="this.getUID()">
```

Reply:
Updated